android/res/values-zh/strings.xml

```diff
@@ -810,10 +810,12 @@
 	<string name="type.barrier.bollard">护柱</string>
 	<string name="type.barrier.border_control">边境检查站</string>
 	<string name="type.barrier.city_wall">城墙</string>
+	<string name="type.barrier.ditch">护城河</string>
 	<string name="type.barrier.entrance">入口</string>
 	<string name="type.barrier.fence">篱笆</string>
 	<string name="type.barrier.gate">门</string>
 	<string name="type.barrier.hedge">树篱</string>
+	<string name="type.barrier.kissing_gate">门</string>
```

Comment:
`kissing gate` don't have a exactly translation name in Chinese, those things are seldom seen in China(This is the first time I know this kind of barrier), I also don't know how to translate it 😥